.github/workflows/test.yml

```diff
@@ -14,16 +20,27 @@ jobs:
     permissions:
       contents: 'read'
 
+    defaults:
+      run:
+        working-directory: testworkflows
+
     steps:
       - name: Clone the code
-        uses: actions/checkout@v5
+        uses: 'actions/checkout@v5'
+
+      - name: Install uv
+        uses: 'astral-sh/setup-uv@v6'
+        with:
+          version: "0.8.17"
+          enable-cache: true
 
-      - name: Setup Go
-        uses: actions/setup-go@v6
+      - name: Setup Python
+        uses: 'actions/setup-python@v6'
         with:
-          go-version-file: go.mod
+          python-version-file: "testworkflows/.python-version"
+
+      - name: Install Dependencies
+        run: uv sync --group dev --group prod
 
-      - name: Running Tests
-        run: |
-          go mod tidy
-          make test
+      - name: Run Tests
+        run: uv run poe test
```

Comment:
Die unit-tests kannst du ruhig zusammen mit dem check task ausführen. Nur e2e in einem separaten Workflow.